app/models/review.rb

```diff
@@ -1,4 +1,5 @@
 class Review < ApplicationRecord
   belongs_to :shelter
+  validates :rating, format: { with: /([1-5])/, message: "can only be between 1-5" }
```

Comment:
This seems weird for edge cases in rails - would we have to do this for every possible field?